docs/CONTRIBUTING.md

```diff
@@ -1,22 +0,0 @@
-# Contributing to xolotl 
-
-`xolotl` is far from feature complete, and your contributions are welcome. 
-
-## Reporting bugs
-
-* Is it a bug? Are you sure the bug persists after you run `transpile` and `compile`? 
```

Comment:
Is it a bug? Are you sure the bug persists after you run `transpile` and `compile` and `xolotl.cleanup`?

Reply:
done